src/ontology/mondo-edit.obo

```diff
@@ -140186,7 +140186,7 @@ is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditar
 
 [Term]
 id: MONDO:0007039
-name: neurofibromatosis type 2
```

Comment:
This previous name should be added as an exact synonym so that users can still find this term using this name. (note that there should be an x-ref for this exact synonym)